CHANGELOG.md

```diff
@@ -8,6 +8,13 @@ librdkafka v2.5.0 is a feature release.
     [v1.9.4](https://github.com/lz4/lz4/releases/tag/v1.9.4), which contains
     bugfixes and performance improvements (#4726).
 
+## Fixes
+
+### General fixes
+
+  * Fix segfault when using long client id because of erased segment when using flexver. (#4689)
```

Comment:
You can keep a long description here with the `Issues` and `Happens since` sections and put a one line description in the summary thar goes below `librdkafka v2.5.0 is a feature release`.

```suggestion
  *  Issues: confluentinc/confluent-kafka-dotnet#2084
     Fix segfault when a segment is erased and more data is written to the buffer.
     Happens since 1.x when a portion of the buffer (segment) is erased for flexver or compression.
     More likely to happen since 2.1.1, because of the upgrades to flexver, with certain string sizes like a long client id (#4689).
```

Reply:
Thanks. Updated